cypress/integration/common/steps.js

```diff
@@ -35,13 +35,38 @@ const annoyingParams = {
 };
 
 Given("I am logged in", () => {
-  cy.login(loginCredentials);
+  cy.neode()
+    .first("User", {
+      name: narratorParams.name
+    })
+    .then(async user => {
```

Comment:
Doesn't that lead to warning of cypress complaining about using promises?

Reply:
maybe, but honestly we'd have to refactor the whole cypress setup since we have these in many of the factories